test-requirements.txt

```diff
@@ -1,6 +0,0 @@
-mock
```

Comment:
That's right. Get ride of test-requirements.txt because it should be requirements.txt for natural ordering!